### DISCUSSION TITLE
Documentation for abbreviations

### DISCUSSION THREAD

**Discussion by TheSeanFraser** [bmvandoren/Nighthawk#14] [Ideas]
I've been getting a few results with four letter codes that are not banding codes - for example "SWLI". Is there documentation for these abbreviations?

**Comment by Mattk70** [comment]
I was wondering the same thing and then I found this, which was marvellous: https://beki.jamessw.com/nfc.

I'll paste the info here, in case the site goes poof.

 BLUEB #

These burbs are bluebs.

Eastern Bluebird (EABL)
Western Bluebird (WEBL)
Mountain Bluebird (MOBL)

BUNT #

Buntings (and a Grosbeak!?)

Indigo Bunting (INBU)
Blue Grosbeak (BLGR)
Painted Bunting (PABU)
Lazuli Bunting (LAZB)
Varied Bunting (VABU)

BZWA #

BuZzing WArblers. (OldBird reference)

Chestnut-sided Warbler (CSWA)
Kirtland's Warbler (KIWA)
Hooded Warbler (HOWA)
Common Yellowthroat (COYE)

CCBRS #

From the bird names

Clay-colored Sparrow (CCSP)
Brewer's Sparrow (BRSP)

CUPS #

It looks like a cup.

Chipping Sparrow (CHSP)
American Tree Sparrow (ATSP)

DBUP #

Double-Banded UP-seep

Tennessee Warbler (TEWA)
Nashville Warbler (NAWA)
Orange-crowned Warbler (OCWA)
Black-throated Green Warbler (BTNW)
Golden-cheeked Warbler (GCWA)
Hermit Warbler (HEWA)
Townsend's Warbler (TOWA)
Lucy's Warbler (LUWA)
Virginia's Warbler (VIWA)
Colima Warbler (COLW)
Grace's Warbler (GRWA)
Black-throated Gray Warbler (BTYW)

DESP #

Descending Sparrows (OldBird reference)

Field Sparrow (FISP)
Nelson's Sparrow (NESP)
Saltmarsh Sparrow (SALS)
LeConte's Sparrow (LCSP)
Henslow's Sparrow (HESP)
Bachman's Sparrow (BACS)
Savannah Sparrow (SAVS)

DEWA #

Descending Warblers (OldBird reference)

Yellow-throated Warbler (YTWA)
Northern Parula (NOPA)
Pine Warbler (PIWA)

GCBI #

From the bird names

Gray-cheeked Thrush (GCTH)
Bicknell's Thrush (BITH)

GROS #

Grosbeaks

Rose-breasted Grosbeak (RBGR)
Black-headed Grosbeak (BHGR)

HSSP #

Grasshopper Sparrow (GRSP)
Vesper Sparrow (VESP)
White-crowned Sparrow (WCSP)
Seaside Sparrow (SESP)

MWAR #

From the bird names

MacGillivray's Warbler (MGWA)
Mourning Warbler (MOWA)

SBUF #

Single-banded up or flat?

Yellow-rumped Warbler (YRWA)
Palm Warbler (PAWA)
Black-throated Blue Warbler (BTBW)
Prothonotary Warbler (PROW)
Swainson's Warbler (SWWA)
Ovenbird (OVEN)
Blue-winged Warbler (BWWA)
Golden-winged Warbler (GWWA)

SFHS #

Song, Fox, Harris Sparrows (and a couple others)

Fox Sparrow (FOSP)
Song Sparrow (SOSP)
Harris's Sparrow (HASP)
Golden-crowned Sparrow (GCSP)
White-throated Sparrow (WTSP)

SWLI #

From the bird names

Swamp Sparrow (SWSP)
Lincoln's Sparrow (LISP)

TANA #

TANAgers

Scarlet Tanager (SCTA)
Summer Tanager (SUTA)
Western Tanager (WETA)

THSH #

THruSHes

Veery (VEER)
Swainson's Thrush (SWTH)
Wood Thrush (WOTH)
Hermit Thrush (HETH)

WITH #

Dusky Thrush (DUTH)
Naumann's Thrush (NATH)

ZEEP #

Onomatopoeia (OldBird reference)

Northern Waterthrush (NOWA)
Louisiana Waterthrush (LOWA)
Kentucky Warbler (KEWA)
Magnolia Warbler (MAWA)
Bay-breasted Warbler (BBWA)
Blackburnian Warbler (BLBW)
Blackpoll Warbler (BLPW)
Connecticut Warbler (CONW)
Yellow Warbler (YEWA)
Cerulean Warbler (CERW)
Worm-eating Warbler (WEWA)
Cape May Warbler (CMWA)
Red-faced Warbler (RFWA)

**Reply by jamesnw** [reply]
I've learned or gotten confirmation on a couple more, and updated the link above.
HSSP - High sibilant sparrow
SBUF- Single-banded up/flat warblers 
WITH- White's Thrush complex
